simulators/main/simulators.py

```diff
@@ -86,21 +92,22 @@ def is_first_time(self, task):
                 task.next_run = datetime.timestamp(datetime.utcnow()) + 1
                 self.machine.first_time = False
 
-    def is_in_productionTime(self):
-        # if there are no shiftplans for a device, it should not be affected by production-time
-        has_no_shiftplan = True
-        for shiftplan in self.machine.shiftplans:
-            if shiftplan.locationId == self.machine.locationId:
-                has_no_shiftplan = False
-                shiftplan_info = oeeAPI.get_shiftplan_status(self.machine.locationId)
-                if shiftplan_info:
-                    if shiftplan_info.get('status') == "PRODUCTION":
-                        return True
-        return has_no_shiftplan
+    def is_in_production_time(self):
+        if self.machine.locationId:
+            shiftplan_status = oeeAPI.get_shiftplan_status(self.machine.locationId)
+            if shiftplan_status:
+                if shiftplan_status.get('status') == "PRODUCTION":
+                    return True
+            log.debug(f'Can not get the status of machine {self.machine.model.get("label")}, it is out of production time by default')
```

Comment:
if the 'status' field is not equals "PRODUCTION", the debug message will be logged which is misleading b/c the status was successfully retrieved; so I think we should change lines 99+100 to:
`return shiftplan_status.get('status') == "PRODUCTION"` (if that is possible with python syntax)